css/badge.css

```diff
@@ -0,0 +1,123 @@
+:root {
+  --rb-primary: #d97706;
+  --rb-dark-grey: #9ca3af;
+  --rb-green: #10b981;
+  --rb-yellow: #fbbf24;
+  --rb-very-light-grey: #f9fafb;
+  --rb-light-black: #111827;
+}
+
+.badge {
+  padding: 0.2rem 0.4rem;
+  font-weight: 600;
+  border-radius: 5px;
+  letter-spacing: 2px;
+  color: var(--rb-very-light-grey);
+}
+
+.badge.primary {
+  background-color: var(--rb-primary);
+}
+
+.badge.secondary {
+  background-color: var(--rb-dark-grey);
+  color: var(--rb-very-light-grey);
+}
+
+.badge.success {
+  background-color: var(--rb-green);
+  color: var(--rb-very-light-grey);
+}
+
+.badge.warning {
+  background-color: var(--rb-yellow);
+  color: var(--rb-light-black);
+}
+
+.badge-icon {
+  position: relative;
+}
+
+.badge-i {
+  position: absolute;
+  top: 0;
+  right: -0.5rem;
+  font-size: 1.1rem;
+  padding: 2px 0.6rem;
+  z-index: 2;
```

Comment:
z-index doesn't seem necessary here. If it is ignore my comment. According to me, absolute position is enough.

Reply:
Solved